irohad/ametsuchi/impl/wsv_restorer_impl.cpp

```diff
@@ -136,7 +137,13 @@ namespace {
         }
       }
 
-      if (not validator.validateAndApply(std::move(block), *mutable_storage)) {
+      // flush every 1000 blocks
```

Comment:
I think not 1000, but reindex_blocks_flush_cache_size_in_blocks_ blocks

Reply:
yeah fixed